Autopilot/AttitudeManager/Inc/attitudeStateClasses.hpp

```diff
@@ -43,6 +45,7 @@ class sensorFusionMode : public attitudeState
         sensorFusionMode& operator =(const sensorFusionMode& other);
         static SFOutput_t _SFOutput;
 };
```

Comment:
Build fails because of this stuff, I think. Don't comment this state. Rather just create a dummy implementation for sensor fusion_getResult that does nothing. Commenting out states fucks with the state control logic and breaks the unit tests.